PayTests/PaySessionTests.swift

```diff
@@ -31,20 +31,30 @@ import PassKit
 @available(iOS 11.0, *)
 class PaySessionTests: XCTestCase {
     
+    private let shopName = "Jaded Labs"
+    
     // ----------------------------------
     //  MARK: - Init -
     //
     func testInit() {
         let checkout = Models.createCheckout()
         let currency = Models.createCurrency()
-        let session  = PaySession(checkout: checkout, currency: currency, merchantID: "some-id")
+        let session  = PaySession(shopName: shopName, checkout: checkout, currency: currency, merchantID: "some-id")
         
         XCTAssertEqual(session.merchantID, "some-id")
         XCTAssertEqual(session.currency.countryCode, currency.countryCode)
         
         XCTAssertFalse(session.identifier.isEmpty)
     }
     
+    func testDeprecatedInit() {
+        let checkout = Models.createCheckout()
+        let currency = Models.createCurrency()
+        let session  = PaySession(checkout: checkout, currency: currency, merchantID: "some-id")
+        
+        XCTAssertEqual(session.shopName, "TOTAL")
```

Comment:
Is it valid? Shouldn't it be `Jaded Labs`

Reply:
Oh, never mind found

Reply:
Yeah, it the old version.